src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/StegService.kt

```diff
@@ -63,13 +63,12 @@ class StegService(
             val sisteBehandling = behandlingService.hentSisteBehandlingSomErIverksatt(fagsakId = behandling.fagsak.id)
                                   ?: behandlingService.hentSisteBehandlingSomIkkeErHenlagt(behandling.fagsak.id)
                                   ?: throw Feil("Forsøker å opprette en revurdering eller teknisk opphør, men kan ikke finne tidligere behandling på fagsak ${behandling.fagsak.id}")
-            val barnFraSisteBehandling =
-                    personopplysningGrunnlagRepository.findByBehandlingAndAktiv(sisteBehandling.id)?.barna?.map { it.personIdent.ident }
-                    ?: throw Feil("Forsøker å opprette en revurdering eller teknisk opphør, men kan ikke finne personopplysningsgrunnlag på siste behandling ${behandling.id}")
+
+            val barnFraSisteBehandlingMedAndelTilkjentYtelse = behandlingService.finnBarnFraBehandlingMedTilkjentYtsele(sisteBehandling.id)
```

Comment:
```suggestion
            val barnFraSisteBehandlingMedUtbetalinger = behandlingService.finnBarnFraBehandlingMedTilkjentYtsele(sisteBehandling.id)
```

Kanskje lettere å forstå utbetalinger 🤷‍♂️